_posts/0000-01-02-Derek-Emsbach.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
A third slide is coming right at you! 
![image](https://user-images.githubusercontent.com/37425403/51727159-ef2c9300-2027-11e9-9585-dbe206fdccec.png)